cmd/kubeless/ingressCreate.go

```diff
@@ -88,13 +88,13 @@ var ingressCreateCmd = &cobra.Command{
 
 		err = utils.CreateIngress(client, f, ingressName, hostName, ns, enableTLSAcme)
 		if err != nil {
-			logrus.Fatalf("Can't create ingress route: %v", err)
+			logrus.Fatalf("Can't create route: %v", err)
 		}
 	},
 }
 
 func init() {
 	ingressCreateCmd.Flags().StringP("hostname", "", "", "Specify a valid hostname for the function")
 	ingressCreateCmd.Flags().StringP("function", "", "", "Name of the function")
-	ingressCreateCmd.Flags().BoolP("enableTLSAcme", "", false, "If true, Ingress will be configured for use with kube-lego")
+	ingressCreateCmd.Flags().BoolP("enableTLSAcme", "", false, "If true, routing rule will be configured for use with kube-lego")
```

Comment:
would this option make sense if the `route` is not `ingress`? maybe we should create a `ingress` subcommand or have a `--type=ingress` parameter to provide advanced options in case it is needed.

Reply:
I am planning to fix this as part of #475, #476. As incremetnal change let the renaming go in this PR.

Reply:
@murali-reddy so you meant we can cancel this PR because this is part of your PRs for these issues ? I am fine with both options. Actually this PR is just small changes.

Reply:
No, i menat since this PR is just name change, once you merge, i will cleanup and address other issues like the one raised above.

Reply:
👍